components/Heading/index.js

```diff
@@ -14,11 +14,12 @@ const defaults = {
   h4: 's'
 };
 
-const Heading = ({ as, size, children }) => {
+const Heading = ({ as, size, children, id }) => {
   return React.createElement(
     as,
     {
-      className: sizes[size || defaults[as]]
+      className: sizes[size || defaults[as]],
+      id
```

Comment:
This is just a heading component, so I think you would call it like this:
`<Heading as="h2"></Heading>`
or
`<Heading size="l"></Heading>`

This is so that all the headings are consistent across the app and it's also easier to write than
`<h2 className="govuk-heading-l"></h2>`
This isn't something I wrote, I think this is leftover code from vulnerabilities

Reply:
I see, thanks for clarifying!